varnish/tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+import os
+import subprocess
+import time
+
+import common
+
+
+@pytest.fixture
+def aggregator():
+    from datadog_checks.stubs import aggregator
+    aggregator.reset()
+    return aggregator
+
+
+@pytest.fixture(scope="session")
+def spin_up_varnish():
+    env = os.environ
+    target = "varnish{}".format(env["VARNISH_VERSION"].split(".")[0])
+    args = [
+        "docker-compose",
+        "-f", os.path.join(common.HERE, 'compose', 'docker-compose.yaml')
+    ]
+    subprocess.check_call(args + ["down"], env=env)
+    subprocess.check_call(args + ["up", "-d", target], env=env)
+    time.sleep(2)
```

Comment:
Can we find a better way to wait for the service to be ready?

Reply:
Hmm I thought about it, and couldn't find a good way. The docker logs are useless, and the container is up really quickly. If I don't put the `sleep`, it _almost_ works: there is only a issue with the first test where for some reason there is an extra line in the output which breaks the xml parsing.